roles/openshift_logging_elasticsearch/templates/operator_adopt.j2

```diff
@@ -0,0 +1,79 @@
+apiVersion: "elasticsearch.redhat.com/v1alpha1"
+kind: "Elasticsearch"
+metadata:
+  name: "{{ es_cluster_name }}"
+spec:
+  securityConfig:
+    certificatesSecret: " {{ es_secret_name }}"
```

Comment:
there is extra-space, it'll try to look for secret where first character is whitespace

Reply:
it shouldn't have the extra whitespace, i updated the template to remove that